### PR TITLE
CM-301:Get Item by Index from Dictionary in C#

### PR DESCRIPTION
**Pull request by ArslanAhmet**
First Commit

### CODE REVIEW

File: collections-csharp/DictionaryGetValueByIndex/DictionaryGetValueByIndex/Program.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿
+using System.Text;
+
+namespace DictionaryGetValueByIndex
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var capitals = new Dictionary<string, string>(){
```
Comment (CodeMazeBlog) [inline collections-csharp/DictionaryGetValueByIndex/DictionaryGetValueByIndex/Program.cs:10]: Strange bracket formatting. Doesn't VS fix this automatically for you?

File: collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace DictionaryTests
+{
+    [TestClass]
+    public class DictionaryGetValueByIndexTests
+    {
+        [TestMethod]
+        public void GivenDictionary_WhenAccessedByIndex_ThenReturnElement()
+        {
+            var capitals = new Dictionary<string, string>(){
```
Comment (CodeMazeBlog) [inline collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs:13]: Formatting.

File: collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            var element = capitals.ElementAt(2);
+
+            Assert.AreEqual( "USA", element.Key);
+        }
+
+
+        [TestMethod]
+        public void GivenDictionary_WhenAccessedByIndex_ThenReturnElementValue()
+        {
+            var capitals = new Dictionary<string, string>(){
```
Comment (CodeMazeBlog) [inline collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs:28]: Formatting

File: collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                {"Turkey", "Ankara"},
+                {"UK", "London"},
+                {"USA", "Washington"}
+            };
+
+            var element = capitals.ElementAt(2);
+
+            Assert.AreEqual( "USA", element.Key);
```
Comment (CodeMazeBlog) [inline collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs:21]: Formatting.

**Review by CodeMazeBlog — CHANGES_REQUESTED** [review]
A few formatting issues. Try to let VS fix those.

File: collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs (hunk 4 above)
Comment (ArslanAhmet) [reply collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs:22]: What is the formatting problem, I can not get it.

**Pull request merged**